app/api/controllers/section.py

```diff
@@ -46,6 +46,14 @@ def post(self):
         return map_to_dto(section), 201
 
 
+@section_ns.route("/all")
+class AllSections(Resource):
+    def get(self):
+        sections = SectionModel.query.all()
```

Comment:
Instead of accessing the SectionModel here in api, you should add a method in the api/dao/section_dao.py to get all sections from the DB. 

The DAO pattern defines the SectionDAO class with methods for interacting with the user model in the database.

Reply:
@ApheleiaS A similar pattern is followed for GET all categories endpoint. So for consistency, I used the similar approach.

Reply:
I think we need to recode some part of the code, we have inconsistency in a few places. Also, I think we don't need `all` in the endpoint URL, it is redundant: `sections` by itself is enough (it is a data resource endpoint, so it is a collection actually).